examples/all-clusters-app/all-clusters-common/src/energy-evse-stub.cpp

```diff
@@ -23,14 +23,24 @@ using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::EnergyEvse;
 
 static std::unique_ptr<EnergyEvseDelegate> gDelegate;
+static std::unique_ptr<EvseTargetsDelegate> gEvseTargetsDelegate;
 static std::unique_ptr<EnergyEvseManager> gInstance;
 
 void emberAfEnergyEvseClusterInitCallback(chip::EndpointId endpointId)
 {
     VerifyOrDie(endpointId == 1); // this cluster is only enabled for endpoint 1.
```

Comment:
Why does this only work for ep1? Seems like this will cause problems for people down the line

Reply:
The cluster does work on any endpoint, but this is in the all-clusters-app so I was following the pattern used in other all-clusters stubs where only EP 1 is used. In Energy-Management-App it is created using code (no ember) and can use any EP.

I can remove it, but the convention from other stubs in all-clusters seem to have this VerifyOrDie

Reply:
Marking this as resolved.